### PR TITLE
MAT-6193: QDM Test Case Coverage Tabs and Accordion

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-6193](https://jira.cms.gov/browse/MAT-6193)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/testCaseLanding/qdm/TestCaseCoverage/TestCaseCoverage.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import "twin.macro";
+import "styled-components/macro";
+import CoverageTabList from "./CoverageTabs/CoverageTabList";
+
+const TestCaseCoverage = ({ groupPopulations, measureCql }) => {
+  return (
+    <div tw="p-5" style={{ paddingRight: ".25rem" }}>
+      <CoverageTabList
+        data-testid="coverage-tab-list"
+        groupPopulations={groupPopulations.populations.filter(
```
Comment (adongare) [inline src/components/testCaseLanding/qdm/TestCaseCoverage/TestCaseCoverage.tsx]: I think this is going to fail if there are no groups

File: src/components/testCaseLanding/qdm/TestCaseList.tsx [hunk 2]
```diff
@@ -406,11 +395,16 @@ const TestCaseList = (props: TestCaseListProps) => {
...
-            {activeTab === "coverage" && coverageHTML && (
-              <CodeCoverageHighlighting
-                coverageHTML={coverageHTML[selectedPopCriteria.id]}
-              />
+            {activeTab === "coverage" && (
+              <div tw="overflow-x-auto sm:-mx-6 lg:-mx-8">
+                <div tw="py-2 inline-block min-w-full sm:px-6 lg:px-8">
+                  <TestCaseCoverage
+                    data-testid="test-case-coverage"
+                    groupPopulations={selectedPopCriteria}
```
Comment (adongare) [inline src/components/testCaseLanding/qdm/TestCaseList.tsx]:
```suggestion
                    group={selectedPopCriteria}
```
or 

```suggestion
                    populationCriteria={selectedPopCriteria}
```

File: src/util/GroupCoverageHelpers.ts [hunk 3]
```diff
@@ -69,3 +69,23 @@ export const mapCql = (measureCql: string, measureGroups): MappedCql => {
     }, {});
   }
 };
+
+export const mapCoverageCql = (measureCql: string, groupPopulations) => {
```
Comment (adongare) [inline src/util/GroupCoverageHelpers.ts:73]: nice to have a couple of unit tests for this function

**Review by adongare — APPROVED** [review]
